containers/clang17/Makevars

```diff
@@ -7,7 +7,7 @@ CXX17=clang++-17 -stdlib=libc++
 CXX20=clang++-17 -stdlib=libc++
 CXX23=clang++-17 -stdlib=libc++
 FC=flang-new-17
-
+LIBS=-lFortran_main -lFortranRuntime -lFortranDecimal -lm
```

Comment:
Don't we only need these for for packages with Fortran code? I.e. in `FLIBS`?

Reply:
Sorry typo, you are right